package/kup/Makefile

```diff
@@ -0,0 +1,57 @@
+.PHONY: default all clean build install          \
```

Comment:
Please regenerate this project with [cruft](https://cruft.github.io/cruft/):

```
cruft create https://github.com/runtimeverification/python-project-template.git
```

This way, we can easily apply template changes.

Reply:
i used cookiecutter. is that not used any more? if not, please update the repo.

Reply:
`cruft` uses `cookiecutter`, it just adds some versioning capabilities.

Reply:
i see, still, the readme needs updating in the `python-project-template` repo